capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,67 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class UniversityData {
+
+  public static UniversityData create(
+      String name,
+      String city,
+      float latitude,
+      float longitude,
+      int flagMainCampus,
```

Comment:
what does `flag` mean in this name? can this be better represented as a boolean? or a enum (if there are other types of campuses)?